uiowa/uiowa
# Added margin top and margin bottom layout builder styles to Collection block

See: https://github.com/uiowa/uiowa/issues/4502

# How to test

Pull down this branch and import the config changes, create a new Collection block and add the margin top and margin bottom styles and confirm they're applied.
